controllers/default_controller.py

```diff
@@ -21,22 +21,23 @@ def systemGetAll(host = None, user = None, password = None):
             units = []
             components = handle.query_children(in_dn="sys", class_id=x["ciscoXmlName"])
```

Comment:
This PR is only for the enabling of hound.  This issue will be addressed as part of [RAC-4150](https://rackhd.atlassian.net/browse/RAC-4150)